src/configuration/SSH/OpenSSH/7.3/sshd_config

```diff
@@ -0,0 +1,118 @@
+#-----------------------------
+# Daemon
+#-----------------------------
+
+# Specifies the port number that sshd(8) listens on.
+# The default is 22.
+#Port 1022
+
+# Specifies the local addresses sshd(8) should listen on.
+# The default is to listen on all local addresses.
+#ListenAddress 192.168.23.42
+
+# Specifies a file containing a private host key used by SSH.
+# The default is
+## /etc/ssh/ssh_host_dsa_key,
+## /etc/ssh/ssh_host_ecdsa_key,
+## /etc/ssh/ssh_host_ed25519_key and
+## /etc/ssh/ssh_host_rsa_key for protocol version 2.
+HostKey /etc/ssh/ssh_host_ed25519_key
+HostKey /etc/ssh/ssh_host_rsa_key
+
+# Specifies the maximum number of concurrent unauthenticated connections
+# to the SSH daemon.
+# The default is 10:30:100.
+MaxStartups 3
+
+# Specifies the maximum number of authentication attempts permitted per
+# connection.
+# The default is 6.
+MaxAuthTries 1
```

Comment:
> If you don't authenticate successfully on first try, you're using it wrong.
> 
> I have been running this setting for many years and after an initial hiccup with some clients like on OS X that take all local SSH-pubkeys and iterate through them, found no more problems. If you set up your client correctly, this is what you want.

That essentially comes back to the discussion on password-less authentication. See below.